kotlin-grpc/build.gradle.kts

```
@@ -0,0 +1,9 @@
+dependencies {
+    implementation(project(":kotlin"))
+    implementation(project(":grpc"))
+
+    // kotlin
+    implementation(libs.kotlin.coroutines.jdk8)
```

We all know it's Kotlin from `libs.kotlin`

```suggestion
    implementation(libs.kotlin.coroutines.jdk8)
```

---

In 2c31706, I remove unused comments!